web/themes/custom/bixal_uswds/src/sass/drupal-system/cookies.scss

```diff
@@ -0,0 +1,55 @@
+// Based on this guidance:
+// https://github.com/jfeltkamp/cookiesjsr/blob/master/README.md#styling
```

Comment:
That's pretty awesome that they make it so easy to set these values. Thanks for updating the CSS.